arm/Microsoft.CognitiveServices/accounts/.parameters/parameters.json

```diff
@@ -24,6 +24,15 @@
                 }
             ]
         },
+        "networkAcls": {
+            "value": {
+                "defaultAction": "deny",
+                "virtualNetworkRules": []
+            }
+        },
```

Comment:
Why this?

Reply:
Came out of a discussion with Marius to test if it works. I.e. can I set network rules without specifying the entire object.

Reply:
I see, I'd probably use different tests to test the networking and instead have the private endpoint parameters in the main parameter.json file. What do you think?

Reply:
The reason it is in a different file is that the Face API (currently used in the `parameters.json`) does not support private endpoints. Should I remove that test and only deploy the speech service?

Reply:
Then how about renaming parametrs.json to face.parameters.json?
Back to the original comment my point is more generally about the fact that since we reference "examples" in the readme I'd expect them to make sense and not only test the feasibility of each combination. However, I wouldn't block this PR for this comment.

Reply:
ok, then let me test adding a network rule. I guess that would be a valid scenario.

Reply:
I can rename it to `face.` by the way - though then we don't have an obvious parameter file for tests like 'diagnostic settings' anymore :D

Reply:
Updated and re-running tests

Reply:
done & green

Reply:
Wouldn't you see that similar to what we have for web sites and vms? we don't have a parameters.json there but the naming is following the specific use case. If we wanted the same here, we should have:

- face.parameters.json
- speech.parameters.json
- speech.min.parameters.json
- speech.encr.parameters.json

It's not strictly related to this PR so again I don't want to block it, but we should agree on some consistency for the parameter file naming

Reply:
Opened issue #1564 to continue the conversation there and to not block this PR